types/data-formats/output-data.d.ts

```diff
@@ -3,15 +3,15 @@ import {BlockToolData} from '../tools';
 /**
  * Output of one Tool
  */
-export interface OutputBlockData {
+export interface OutputBlockData<S extends string = string, T extends object = any> {
   /**
-   * Too type
+   * Tool type
    */
-  type: string;
+  type: S;
```

Comment:
The type is always a string. Are you sure we need a generic here?

Reply:
By using Generics, Editor.js users can describe string literal types. 

https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types

By using string literal types in `OutputBlockData['type']`, Editor.js users can describe more type-safe `OutputData` in each application. 

See also: 
https://github.com/hata6502/editorjs-examples/blob/main/examples/ExampleEditorJSData.ts#L17